### PR TITLE
SITL:make MAG param consistent names

### PR DESCRIPTION
**Pull request by Hwurzburg**
all other first instance GPS names are just GPS

### CODE REVIEW

File: libraries/SITL/SITL.cpp [hunk 1]
```diff
@@ -455,12 +455,12 @@ const AP_Param::GroupInfo SIM::var_mag[] = {
...
     AP_GROUPINFO("MAG_DELAY",      3, SIM,  mag_delay, 0),
-    AP_GROUPINFO("MAG_OFS",        4, SIM,  mag_ofs[0], 0),
+    AP_GROUPINFO("MAG1_OFS",        4, SIM,  mag_ofs[0], 0),
     AP_GROUPINFO("MAG_ALY",        5, SIM,  mag_anomaly_ned, 0),
     AP_GROUPINFO("MAG_ALY_HGT",    6, SIM,  mag_anomaly_hgt, 1.0f),
-    AP_GROUPINFO("MAG_DIA",        7, SIM,  mag_diag[0], 0),
-    AP_GROUPINFO("MAG_ODI",        8, SIM,  mag_offdiag[0], 0),
-    AP_GROUPINFO("MAG_ORIENT",     9, SIM,  mag_orient[0], 0),
+    AP_GROUPINFO("MAG1_DIA",        7, SIM,  mag_diag[0], 0),
+    AP_GROUPINFO("MAG_1ODI",        8, SIM,  mag_offdiag[0], 0),
```
Comment (tridge) [inline libraries/SITL/SITL.cpp:462]: type, should be MAG1_ODI

**Review by tridge — CHANGES_REQUESTED** [review]

File: libraries/SITL/SITL.cpp (hunk 1 above)
Comment (Hwurzburg) [reply libraries/SITL/SITL.cpp]: already caught that and pushed fix

**Review by peterbarker — APPROVED** [review]
LGTM

**Comment by tridge** [comment]
there are some tests in Tools/autotest/common.py that need updating for the new name

**Review by tridge — APPROVED** [review]

**Pull request merged**